File: sb/src/main/java/dev/muldev/appgestiongym/Memberships/Domain/Membership.java
package dev.muldev.appgestiongym.memberships.domain;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@NoArgsConstructor
@AllArgsConstructor
@Data
public class Membership {
    private Integer idmatricula;
    private Integer idcliente;
    private Integer idtarifa;
    private Date fechaAlta;
    private Date fechaAbonadoHasta;
}

File: angular/src/app/dto/membership.ts
export class Membership {
    idmatricula: number;
    idcliente: number;
    idtarifa: number;
    fechaAlta: Date;
    fechaAbonadoHasta: Date;

    public constructor(init?: Partial<Membership>) {
        Object.assign(this, init);
    }

}
